vars/dockerExecuteOnKubernetes.groovy

```diff
@@ -423,7 +423,8 @@ private List getContainerEnvs(config, imageName) {
     }
 
     if (dockerEnvVars) {
-        for (String k : dockerEnvVars.keySet()) {
+        for (int i = 0; i < dockerEnvVars.keySet().size(); i++) {
```

Comment:
what about using `dockerEnvVars.each()`?

That should be pretty safe according to https://jenkins.io/blog/2017/09/08/enumerators-in-pipeline/

Reply:
Considering `each` as being safe, therefore refactored and updated.